src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
If your main concern is that verifier won't run and bad packages will get out, how about we do something simpler:
 - Pack creates *.nupkg files in an intermediate location, such as `/obj/packages/`
 - VerifyPackages moves *.nupkg files to `/artifacts/build/` if and only if they pass verification steps.

But regarding target chaining, we could use this pattern instead:

```xml
<!-- in Lifecycle.targets -->
<Target Name="Pack" DependsOnTargets="$(PackDependsOn)" />

<!-- in NugetPackageVerifier.props -->
<PropertyGroup>
   <PackDependsOn>$(PackDependsOn);VerifyPackages</PackDependsOn>
</PropertyGroup>

<!-- in NugetPackageVerifier.targets -->
<Target Name="VerifyPackages" />
```